docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -143,7 +143,12 @@ Set any as needed or all of the following environment variables to troubleshoot
 
 ### `SEMGREP_BRANCH`
 
-Set `SEMGREP_BRANCH` to define the branch name for the URL used to generate hyperlinks in the [Findings](/docs/semgrep-code/findings) page. To avoid hardcoding this value, check your CI provider's documentation for available environment variables that can automatically detect the correct values for every CI job. 
+Set `SEMGREP_BRANCH` to define the branch name for the scan, if the branch name is not auto-detected or you want to override it. The branch name is used in the following ways:
+
+* To track findings in the same branch over time
+* To show in which branches a finding was identified (inclukding links to the branch in the [Findings](/docs/semgrep-code/findings) page)
+
+To avoid hardcoding this value, check your CI provider's documentation for available environment variables that can automatically detect the correct values for every CI job. 
```

Comment:
Might want to also say that SCP will automatically normalize the branch names so, for example, `refs/heads/develop` and `develop` will be treated as the same branch (I think this is new clean-up work thanks to @zdhickman)

Reply:
I think there are some other things that all get bucketed together as `<unknown>` but that's pretty much it

Reply:
Those would be strange values for a manual setting, so I think we can safely omit that in this case :)